modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -3066,6 +3066,8 @@ public void testUpdateObjectEntry() throws Exception {
 				"lastName", "o Discípulo Amado"
 			).put(
 				"listTypeEntryKeyRequired", "listTypeEntryKey2"
+			).put(
+				"middleName", () -> null
```

Comment:
@p-albuquerque wasn't the value of this field already `null`? If so, how this test was passing?